qt/aqt/addcards.py

```diff
@@ -7,6 +7,7 @@
 import aqt.editor
 import aqt.forms
 import aqt.modelchooser
+from anki.backend_pb2 import SearchTerm
```

Comment:
On a previous PR, I got the impression that aqt should not call (and import from?) the backend directly. Is that accurate?
Either way, I'd have all modules import from one place: rsbackend, backend_pb2 or collection.

Reply:
That should have been from collection - the IDE added it and I didn't notice when reviewing the patch. Good catch :-)

I wish there were a way to mark rsbackend and backend_pb2 as unimportable from outside pylib. A leading _ might help with automatic completion, but we originally had code importing TR from rsbackend, which we'd break by doing that. :-(